src/common/messages.js

```diff
@@ -87,11 +87,31 @@ const messages = defineMessages({
         description: 'Error message for when an invalid user is entered',
         id: 'boxui.validation.invalidUserError',
     },
+    anonymousUser: {
```

Comment:
I think we generally want to avoid adding to these global files. We should move these new strings to a messages file that can be shared by the renderers.